src/content/guides/packages.md

```diff
@@ -178,6 +178,78 @@ due to conflicting version constraints in the pubspec.
 To identify out-of-date packages that require editing the pubspec,
 use [`dart pub outdated`][outdated].
 
+## Getting dependencies in production - `pub get --enforce-lockfile`
+
+In some cases `dart pub get` will not give you the exact versions locked in
+`pubspec.lock`. For example:
```

Comment:
Does the following list include all possible situations or just some? If it covers the major cases, I think it's probably fine to lead directly into the list:

```suggestion
In some situations, `dart pub get` does not retrieve
the exact package versions locked in the `pubspec.lock` file:
```

Reply:
I think these (together with the hash mismatch) are the major ones.